crates/uv-trampoline/src/bounce.rs

```diff
@@ -509,6 +499,12 @@ pub fn bounce(is_gui: bool) -> ! {
     exit_with_status(exit_code);
 }
 
+#[cold]
+fn error_and_exit(message: &str) -> ! {
+    error!("{}", message);
+    exit_with_status(1);
```

Comment:
Does this pop-up persist beyond the program exiting? Or block exit until it's ack'd?

Reply:
popup blocks exit (or maybe startup? my example was too trivial to distinguish that) yeah.

Reply:
I guess in the error cases that's not even a real distinction (it's a meaningful distinction with warnings, but none of those popup anymore so not a distinction anymore :)

Reply:
genuinely unsure how to trigger an error popup here. as you said before most of them are "uv is broken, wtf" errors

Reply:
I think you'd have to just like `if true` then throw the pop-up to see if it works as intended. Not a big deal, was just wondering if these would actually be visible as intended.

Reply:
good point. confirmed it blocks the process from exiting until the popup is dismissed, so you see it.